Buildings/Controls/OBC/CDL/Integers/SequenceBinary.mo

```diff
@@ -0,0 +1,125 @@
+within Buildings.Controls.OBC.CDL.Integers;
+block SequenceBinary "Output total stages that should be ON"
+
+  parameter Integer nSta(
+    final min=1)
+    "Maximum stages that could be ON";
+  parameter Real minStaOn(
```

Comment:
Can you explain the behavior without ON and OFF as this block (and CDL) has no notion of ON or OFF.